server/ec2spotmanager/migrations/0013_auto_20180911_2232.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
You only need one migration. You can copy these operations into the previous one.